src/app/_components/MarkdownPage.tsx

```diff
@@ -0,0 +1,37 @@
+import type { WithContext, Thing } from 'schema-dts'
+
+import type { PageAttributes } from '@/types/attributesTypes'
+
+import type { SeoMetadata } from '@/schemas/SeoMetadataSchema'
+
+import { MarkdownContent } from '@/components/MarkdownContent'
+import { PageHeader } from '@/components/PageHeader'
+import { StructuredDataScript } from '@/components/StructuredDataScript'
+
+type MarkdownPageProps = {
+  attributes: PageAttributes
+  body: string
```

Comment:
To be changed to `Parameters<typeof ReactMarkdown>[0]['children']` once #793 is merged.